fsw/pc-linux/src/cfe_psp_start.c

```diff
@@ -531,7 +531,7 @@ void CFE_PSP_DisplayUsage(char *Name)
     printf("       %s --reset PO --subtype 1 --cpuid 1 --cpuname CPU1 --scid 32\n", Name);
     printf(" \n");
 
-    exit(1);
+    exit(EXIT_SUCCESS);
```

Comment:
Not sure about this - the old `exit(1)` is an error/fail code, because traditionally `exit(0)` is success on UNIX.  This should probably be `exit(EXIT_FAILURE)` to maintain this - and usually running with bad options should be a failure, not a success.

Reply:
No worries Joe.
Updated.